docs/operating-scylla/admin-tools/scylla-sstable.rst

```diff
@@ -29,7 +29,8 @@ The command syntax is as follows:
    scylla sstable <operation> <path to SStable>
 
 
-You can specify more than one SStable.
+You can specify more than one SSTable. Additionally, the path to SSTable can point to an S3 fully qualified path in the form of s3://bucket-name/prefix/of/your/sstable/sstable-TOC.txt. To use this feature, you need to have AWS credentials set up in your environment. For more information, see :ref:`Configuring AWS S3 access <aws-s3-configuration>`. Additionally, you must ensure the tool is able to load the correct Scylla YAML file, which can be done using the --scylla-yaml-file parameter or by placing the YAML file in one of the default locations the tool checks.
```

Comment:
Reminder to self to follow-up with a dedicated section for scylla.yaml, just like we have for the schema.
This file is now pretty important for the tool and it deserves a section on how the tool tries to find it.